docs/default-theme-config/README.md

```diff
@@ -451,9 +451,9 @@ editLink: false
 
 ## Simple CSS Override
 
-If you wish to apply simple overrides to the styling of the default theme, you can create an `.vuepress/override.styl` file. This is a [Stylus](http://stylus-lang.com/) file but you can use normal CSS syntax as well.
+If you wish to apply simple overrides to the styling of the default theme, you can create a `.vuepress/styles/palette.styl` file. This is a [Stylus](http://stylus-lang.com/) file but you can use normal CSS syntax here too.
```

Comment:
Your PR is on the version 0.x of vuepress doc, and I don't think that `palette.styl`exists on v0

Reply:
+1

Reply:
I see, perhaps I was reading the wrong docs for the version I was using. Can you link to the latest version of this document please?

Reply:
last version of vuepress doc => https://vuepress.vuejs.org/
v0 of vuepress doc = https://v0.vuepress.vuejs.org/